GcloudBackup.ps1

```diff
@@ -24,12 +24,12 @@ try {
 		. $confFile
 	}
 	else {
-		Write-Host 'Configuration file not found, please reinstall!' -fore red -back black
+		Write-Host 'Configuration file not found, please reinstall!' -ForegroundColor Red -BackgroundColor Black
 		exit 1
 	}
 }
 catch {
-	Write-Host "Please, check your configuration file, there's something incorrect in it. " -fore red -back black
+	Write-Host "Please, check your configuration file, there's something incorrect in it. " -ForegroundColor Red -BackgroundColor Black
```

Comment:
In this exception it would be cool to tell the user where the conf file should be, just in case it has several installations for different purposes.